aip/0140.md

```diff
@@ -27,17 +27,22 @@ intuitive, and consistent with one another.
 
 Field names **should** be in correct American English.
 
-Field definitions in protobuf files **must** use `snake_case` names. These
-names are mapped to an appropriate naming convention in JSON and in generated
-code.
-
 Field names **should** clearly and precisely communicate the concept being
 presented and avoid overly general names that are ambiguous. That said, field
 names **should** avoid including unnecessary words. In particular, avoid
 including adjectives that always apply and add little cognitive value. For
 example, a `proxy_settings` field might be as helpful as
 `shared_proxy_settings` if there is no unshared variant.
 
+### Case
+
+Field definitions in protobuf files **must** use `lower_snake_case` names.
+These names are mapped to an appropriate naming convention in JSON and in
+generated code.
+
+Additionally, each word in the field **must not** begin with a number, because
+it creates ambiguity when converting between snake case and camel case.
```

Comment:
> I suspect it's worth thinking of an example where that's tricky and giving the preferred alternative. Has this come up in any real world examples?

Yes, in Ads (of course). They have fields like `quartile_25_thing`, `quartile_50_thing`, etc. Personally, I think they should use a `map<int32, Foo>` for that use case. I actually pointed that out in the linter documentation I proposed in googleapis/api-linter#556, but I was worried that it might be a rabbit hole here.

Reply:
Nope, that's fine.